packages/ui-library/src/elements/tag-input/index.js

```diff
@@ -38,16 +39,13 @@ const Tag = ( { tag, index, disabled = false, onRemoveTag, screenReaderRemoveTag
 
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
-		<div onKeyDown={ handleKeyDown } className="yst-tag-input__tag" { ...props }>
+		<Badge onKeyDown={ handleKeyDown } { ...props }>
 			<span>{ tag }</span>
-			<button
-				className="yst-tag-input__remove-tag"
```

Comment:
You can probably removing the CSS for these classes too?

Reply:
The css for those classes is not in the style.css file in the tag-input folder.

Reply:
Good catch! Done.

Reply:
I ended up reusing them in the Badge